test/functional/specs/Personalization/C5298194.js

```diff
@@ -0,0 +1,36 @@
+import { t } from "testcafe";
+import createFixture from "../../helpers/createFixture";
+import { orgMainConfigMain } from "../../helpers/constants/configParts";
+import createAlloyProxy from "../../helpers/createAlloyProxy";
+import createCollectEndpointAsserter from "../../helpers/createCollectEndpointAsserter";
+
+createFixture({
+  title: "C5298194: Include propositions on every request"
+});
+
+test.meta({
+  ID: "C5298194",
+  SEVERITY: "P0",
+  TEST_RUN: "Regression"
+});
+
+test("Test C5298194: Include propositions on every request", async () => {
+  const collectEndpointAsserter = await createCollectEndpointAsserter();
+  const alloy = createAlloyProxy();
+  await alloy.configure(orgMainConfigMain);
+
+  let response = await alloy.sendEvent({ documentUnloading: true });
+  await collectEndpointAsserter.assertInteractCalledAndNotCollect();
+  await t
+    .expect("propositions" in response)
+    .ok("The 'sendEvent()' response was missing the 'propositions' property");
+  response = null;
+  await collectEndpointAsserter.reset();
+
+  response = await alloy.sendEvent();
```

Comment:
We probably want to call "collect" at some point in this test. Normally setting documentUnloading to true will trigger a collect call, but if the identity isn't established, it will call interact. If you move the `{ documentUnloading: true }` parameter to this line, it will call collect here.